build/Targets/Analyzers.Versions.targets

```diff
@@ -22,44 +22,44 @@
   <!-- Version info for FxCop analyzer packages -->
   <PropertyGroup>
       <!-- Master FxCop analyzer package, which is just a collection of individual analyzer packages below. -->
-      <FxCopAnalyzersSemanticVersion>1.1.0</FxCopAnalyzersSemanticVersion>
+      <FxCopAnalyzersSemanticVersion>1.2.0</FxCopAnalyzersSemanticVersion>
       <FxCopAnalyzersPreReleaseVersion>beta1</FxCopAnalyzersPreReleaseVersion>
 
       <!-- Dependency version range for packages included in the master FxCop analyzer package: "[$(FxCopAnalyzersDependencyMinimumSemanticVersion) - $(FxCopAnalyzersDependencyNextSemanticVersion))" -->
       <!-- Allow any version of dependent packages between minimum (inclusive) and next (exclusive) semantic versions. -->
       <!-- For example, values of minimum = '1.0.0' and next = '1.2.0' will have a dependency range ">= 1.0.0 && < 1.2.0" -->
       <FxCopAnalyzersDependencyMinimumSemanticVersion>1.0.0</FxCopAnalyzersDependencyMinimumSemanticVersion>
-      <FxCopAnalyzersDependencyNextSemanticVersion>1.2.0</FxCopAnalyzersDependencyNextSemanticVersion>
+      <FxCopAnalyzersDependencyNextSemanticVersion>1.3.0</FxCopAnalyzersDependencyNextSemanticVersion>
```

Comment:
Actually, I think we should get rid of the version range as nuget dependency behavior resolves to the lowest acceptable match by default.

Can you get rid of `FxCopAnalyzersDependencyMinimumSemanticVersion` and `FxCopAnalyzersDependencyNextSemanticVersion` and also its consumers up the chain - get rid of [this](https://github.com/dotnet/roslyn-analyzers/blob/master/build/Targets/Analyzers.Imports.targets#L47) property group and delete entire line [here](https://github.com/dotnet/roslyn-analyzers/blob/master/build/Targets/Analyzers.NuGet.Imports.targets#L10), [here](https://github.com/dotnet/roslyn-analyzers/blob/master/build/Targets/Analyzers.NuGet.Imports.targets#L22) and [here](https://github.com/dotnet/roslyn-analyzers/blob/master/build/Targets/Analyzers.NuGet.Imports.targets#L34). Also delete redundant lines 10 and 11 in [FXCop nuget proj file](https://github.com/dotnet/roslyn-analyzers/blob/master/src/FxCop/NuGet/FxCopAnalyzers.NuGet.proj) The FXCop nuspec should just specify version of dependencies to be '[$currentVersion$,]' over [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/FxCop/NuGet/FxCopAnalyzers.nuspec#L10)